setup.py

```diff
@@ -148,6 +152,15 @@ def get_cpp_flags(build_ext):
 
     raise DistutilsPlatformError(last_err)
 
+def get_nvcc_flags():
+    default_flags = ['--std=c++11', '-O3', '-Xcompiler', '-fPIC']
+    cc_list_env = os.environ.get('HOROVOD_BUILD_CUDA_CC_LIST')
```

Comment:
Can you add docs about this flag and its defaults to https://github.com/horovod/horovod/blob/master/docs/install.rst#environment-variables ?

Reply:
I added documentation for this flag. Right now, the default compute capability is being detected by NVCC at build time. Do you think we should add a set of default compute capabilities to target a particular set of GPU architectures in all builds?

Reply:
I think the way you have it now is reasonable.  Thanks for updating the docs!